tests/io_utils.go

```diff
@@ -85,24 +87,34 @@ func CreateSCSIDisk(nodeName string, opts []string) (address string, device stri
 	_, err := ExecuteCommandInVirtHandlerPod(nodeName, args)
 	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "Failed to create faulty disk")
 
-	args = []string{UsrBinVirtChroot, Mount, Proc1NsMnt, "exec", "--", "/usr/bin/lsscsi"}
-	stdout, err := ExecuteCommandInVirtHandlerPod(nodeName, args)
-	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "Failed to find out address of  SCSI disk")
-
-	// Example output
-	// [2:0:0:0]    cd/dvd  QEMU     QEMU DVD-ROM     2.5+  /dev/sr0
-	// [6:0:0:0]    disk    Linux    scsi_debug       0190  /dev/sda
-	lines := strings.Split(stdout, "\n")
-	for _, line := range lines {
-		if strings.Contains(line, "scsi_debug") {
-			line = strings.TrimSpace(line)
-			disk := strings.Split(line, " ")
-			address = disk[0]
-			address = address[1 : len(address)-1]
-			device = disk[len(disk)-1]
-			break
+	EventuallyWithOffset(1, func() error {
+		args = []string{UsrBinVirtChroot, Mount, Proc1NsMnt, "exec", "--", "/usr/bin/lsscsi"}
+		stdout, err := ExecuteCommandInVirtHandlerPod(nodeName, args)
+		if err != nil {
+			return fmt.Errorf("Error executing lsscsi: %s", err.Error())
```

Comment:
We usually use `%v` and pass the error itself.
But just to make sure we do not duplicate info here: Is the `err` returned not including the information about what command failed? (i.e. can we just return the `err` as is?)

Reply:
Yep we have all of this under the err itself from pod exec, done